packages/synapse-react-client/src/lib/containers/download_list_v2/DownloadCartPage.tsx

```diff
@@ -169,7 +167,11 @@ export const DownloadCartPage: React.FunctionComponent<
                         helpUrl="https://help.synapse.org/docs/Downloading-Data-From-the-Synapse-UI.2004254837.html"
                       />
                     </div>
-                    <Typography variant={'body1'}>
+                    <Typography
+                      variant={'body1'}
+                      component={'div'}
```

Comment:
I wanted to just do `component={'ul'}` but apparently we have this style in SWC:
```css
ol, ul {
  margin-bottom: 0px !important;
}
```
so we'll wrap in a div